cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,97 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	var billingAccounts []string
+	cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+
+	// get a list of CloudAccountID
+	for _, account := range cloudAccounts {
+		if account.CloudProviderID() == "aws" && !helper.HasPrefix(billingAccounts, account.CloudAccountID()) {
+			contracts, hasContract := account.GetContracts()
+			var contractString string
+			if hasContract {
+				dimensions := contracts[0].Dimensions()
+				for _, dimension := range dimensions {
+					if ocm.GetDimensionValue(dimension, "four_vcpu_hour") > 0 ||
+						ocm.GetDimensionValue(dimension, "control_plane") > 0 {
+						contractString = " [Contract enabled]"
+					}
+				}
+			}
+			billingAccounts = append(billingAccounts, account.CloudAccountID()+contractString)
+		}
+	}
+
+	if billingAccount == "" && !interactive.Enabled() {
+		if helper.HasPrefix(billingAccounts, awsCreator.AccountID) {
```

Comment:
I like Cristiano's comment in the other PR and should be included.
```
// if a billing account is not provided we will try to use the infrastructure account as default
```

Reply:
Done